be/src/common/CMakeLists.txt

```diff
@@ -22,15 +22,27 @@
 # where to put generated libraries
 set(LIBRARY_OUTPUT_PATH "${BUILD_DIR}/src/common")
 
-add_library(Common STATIC
-  daemon.cpp
-  status.cpp
-  statusor.cpp
-  resource_tls.cpp
-  logconfig.cpp
-  configbase.cpp
-  minidump.cpp
-)
+
+if ("${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86" OR "${CMAKE_BUILD_TARGET_ARCH}" STREQUAL "x86_64")
+  add_library(Common STATIC
```

Comment:
what's the difference?

Reply:
It is to exclude minidump.cpp when the architecture is aarch64. Since it's not a good solution, `target_sources` is used here to  avoid two `add_library`.

Reply:
maybe you can write two directives

```
add_library(Common STATIIC common_cpp_files)

if (x86)
add_library(...)
endif